exonum/src/api/node/public/explorer.rs

```diff
@@ -246,55 +252,112 @@ impl ExplorerApi {
                 ApiError::NotFound(description)
             })
     }
+
     /// Adds transaction into unconfirmed tx pool, and broadcast transaction to other nodes.
     pub fn add_transaction(
+        name: &str,
+        backend: &mut actix_backend::ApiBuilder,
+        service_api_state: ServiceApiState,
+    ) {
+        let (max_message_len, max_payload_len) = get_message_limits(&service_api_state);
+        let index = move |request: HttpRequest| {
+            let state = request.state().clone();
+            let content_length = request
+                .headers()
+                .get(CONTENT_LENGTH)
+                .unwrap()
```

Comment:
It's [possible](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.4) to send a valid post request without Content-Length header. So it this `unwrap` safe (and correct) here?